test/events/failEvent.json

```diff
@@ -0,0 +1,23 @@
+{
+    "event": "issues", 
+    "payload": {
+      "action": "opened",
+      "issue": {
+        "body": "Fix your broken thigns!",
+        "title": "ur thing is broken fix it!!",
+        "user": {
+          "login": "hiimbex"
+        },
+        "labels": []
```

Comment:
Pardon my ignorance of the API, but will this fail if you omit the `labels` part of the payload altogether? Does the API give you an empty array by default?